perceval/utils/postselect.py

```diff
@@ -35,18 +35,32 @@
 
 
 class PostSelect:
+    """PostSelect is a post-selection callable used to filter out unwanted output states. It is designed to be an
```

Comment:
"callable post-selection" au lieu de "post-selection callable" plutôt non ? Je mettrais également "friendly" au lieu de "expressive". De manière générale je trouve la phrase trop longue. Je mettrais :
"PostSelect is a callable post-selection used to filter out unwanted output states. It is designed to be an friendly way of describing how to filter out our output states. It is aiming to represent post-selection logic on a state after detection."
En tout cas je trouve que la deuxième phrase doit être relue plusieurs fois pour être comprise et du coup mériterais d'être coupée en deux.